<?xml version="1.0"?>
<robot name="ferro_sander">
  <link name="tool_origin">
    <visual>
        <geometry>
            <mesh filename="package://pct/data/meshes/ferro_sander.stl"
                scale="1 1 1"/>
        </geometry>
        <material name="ABS" >
            <color rgba="0.5 0.5 0.5 1.0" />
         </material>    
        </visual>
  </link>
</robot>
